src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    background-image: url('/src/images/star.svg');
```

Comment:
The relative path for the background image might cause issues when the SCSS is compiled to CSS and the project structure changes. It's recommended to use a path that starts from the root of the project or a variable that holds the path to the images folder.